indra/newview/llselectmgr.h

```diff
@@ -901,12 +901,13 @@ class LLSelectMgr : public LLEditMenuHandler, public LLSimpleton<LLSelectMgr>
     static void logAttachmentRequest(LLSelectNode* node, void *user_data);
     static void logDetachRequest(LLSelectNode* node, void *user_data);
     static bool confirmDelete(const LLSD& notification, const LLSD& response, LLObjectSelectionHandle handle);
-    static void packAlphaGammaOverride(LLViewerObject* object);
 
     // Get the first ID that matches test and whether or not all ids are identical in selected objects.
     void getFirst(LLSelectGetFirstTest* test);
 
 public:
+    static void packAlphaGammaBypass(LLViewerObject* object);
+
```

Comment:
This method must be `public `because it is called by `LLSelectMgrAlphaGammaBypassFunctor`.